From fa6be19d1106fd4c35b43a36a26961947d70c208 Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Thu, 27 Apr 2023 13:18:20 +0800
Subject: [PATCH 63/68] Revert "drm/virtio: Correct
 drm_gem_shmem_get_sg_table() error handling"

This reverts commit 87c647def389354c95263d6635c62ca0de7d12ca.
---
 drivers/gpu/drm/virtio/virtgpu_object.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/gpu/drm/virtio/virtgpu_object.c b/drivers/gpu/drm/virtio/virtgpu_object.c
index 1681486860019..d4fab3361d2c3 100644
--- a/drivers/gpu/drm/virtio/virtgpu_object.c
+++ b/drivers/gpu/drm/virtio/virtgpu_object.c
@@ -159,7 +159,6 @@ static int virtio_gpu_object_shmem_init(struct virtio_gpu_device *vgdev,
 	shmem->pages = drm_gem_shmem_get_sg_table(&bo->base.base);
 	if (IS_ERR(shmem->pages)) {
 		drm_gem_shmem_unpin(&bo->base.base);
-		shmem->pages = NULL;
 		return PTR_ERR(shmem->pages);
 	}
 
-- 
2.25.1

